generate_stats.py

```diff
@@ -1,44 +1,81 @@
-# This file is the script that users will actually run to generate the full set of statistics for a file
+#This file is the script that users will actually run to generate the full set of statistics for a file
 
 # set the path to find the current installation of pyMOAB
 import sys
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
 from pymoab import core, types
 from pymoab.rng import Range
-
 # import the new module that defines each of the functions
 import dagmc_stats
+import argparse
```

Comment:
import `argparse` in the same grouping with other built-ins, probably after `sys` in this case